### ISSUE TITLE
An error for missing 'System.ValueTuple`8' is not reported

### ISSUE THREAD

**Issue by AlekseyTs** [dotnet/roslyn#11287]
```
        [Fact]
        public void GenericTupleWithoutTupleLibrary_02()
        {
            var source = @"
class C
{
    static void Main()
    {
    }

    static (T1, T2, T3, T4, T5, T6, T7, T8, T9) M<T1, T2, T3, T4, T5, T6, T7, T8, T9>()
    {
        throw new System.NotSupportedException();
    }
}
" + trivial2uple;

            var comp = CreateCompilationWithMscorlib(source, parseOptions: TestOptions.Regular.WithTuplesFeature());
            comp.VerifyDiagnostics(
                // (8,12): error CS0518: Predefined type 'System.ValueTuple`8' is not defined or imported
                //     static (T1, T2, T3, T4, T5, T6, T7, T8, T9) M<T1, T2, T3, T4, T5, T6, T7, T8, T9>()
                Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound, "(T1, T2, T3, T4, T5, T6, T7, T8, T9)").WithArguments("System.ValueTuple`8").WithLocation(8, 12)
                );
        }
```

Observed: No error reported.

Expected:
error CS0518: Predefined type 'System.ValueTuple`8' is not defined or imported

**Comment by AlekseyTs** [comment]
@jcouv Please verify.

**Comment by jcouv** [comment]
I see the unittest is now passing (ie. an error is appropriately reported).

**Comment by AlekseyTs** [comment]
There is a "Verified" label that is supposed to be set to confirm verification.

**Comment by jcouv** [comment]
Marked as Verified. Thanks
